src/mlpack/methods/logistic_regression/logistic_regression_impl.hpp

```diff
@@ -67,28 +67,38 @@ LogisticRegression<MatType>::LogisticRegression(
 }
 
 template<typename MatType>
-template<typename OptimizerType>
-double LogisticRegression<MatType>::Train(const MatType& predictors,
-                                        const arma::Row<size_t>& responses)
+template<typename OptimizerType, typename... CallbackTypes>
+double LogisticRegression<MatType>::Train(
+        const MatType& predictors,
+        const arma::Row<size_t>& responses,
+        CallbackTypes&&... callbacks)
 {
   OptimizerType optimizer;
-  return Train(predictors, responses, optimizer);
+  return Train(
```

Comment:
Pedantic style issue, to you mind to write this as a single line and break after 80 characters? For more information see https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping for more information.